### PR TITLE
Fix for issue: Dynamic Time Conversion Route

### PR DESCRIPTION
**Pull request by whoiskev1n**
This PR fixes issue #7.

### CODE REVIEW

File: routes/miamiTemperature.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    res.json({
+      temperature: temperature.toFixed(2),
+      unit
+    });
+  } catch (error) {
+    res.status(500).send('Failed to fetch temperature');
+  }
+});
```
Comment (chndt) [inline routes/miamiTemperature.js:24]: This entire block can be removed.

File: routes/miamiTemperature.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+router.get('/miami-temperature', (req, res) => {
+  // Implement your logic here
+  res.send('Temperature in Miami');
+});
```
Comment (chndt) [inline routes/miamiTemperature.js:29]: This should be:
```
router.get('/miami-temperature', async (req, res) => {
  res.send(await fetchMiamiTemperature());
});
```

File: routes/miamiWeather.js
Comment (chndt) [inline routes/miamiWeather.js:1]: This file is empty - let's just remove it.

File: log.json
Comment (chndt) [inline log.json:1]: Can we please exclude this file from version control?

**Review by chndt — CHANGES_REQUESTED** [review]
Really cool - just a few minor remarks :)

**Comment by whoiskev1n** [comment]
Requested changes have been applied. Please review the modifications.

**Pull request merged**